app/views/shared/_search_context_sidebar.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="panel panel-default">
+  <h3 class="sr-only panel-title">Search context</h3>
+  <div class="panel-body">
+    <h3><%= t('blacklight.search.zero_results.search_context') %></h3>
+    <div class="media">
+      <div class="media-body">
+        <%= link_to_catalog_search unless controller_name == 'catalog'%>
```

Comment:
If there's an `article_search?` helper, shouldn't there also be a `catalog_search?` helper too?  Would it ever be the case that both  are true?  Could you do:
```erb
<% if article_search? %>
  <%= link_to_catalog_search %>
<% else %>
  <%= link_to_article_search %>
<% end %>
```

Reply:
✅ good point. I replaced this with your suggested syntax.